test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcTests.server.cs

```diff
@@ -114,6 +117,189 @@ public void GrpcAspNetCoreInstrumentationAddsCorrectAttributes(bool? enableGrpcA
             Assert.StartsWith("grpc-dotnet", activity.GetTagValue(SemanticConventions.AttributeHttpUserAgent) as string);
         }
 
+        // Tests for v1.21.0 Semantic Conventions for database client calls.
+        // see the spec https://github.com/open-telemetry/semantic-conventions/blob/main/docs/rpc/rpc-spans.md
+        // This test emits the new attributes.
+        // This test method can replace the other (old) test method when this library is GA.
+        [Theory]
+        [InlineData(null)]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void GrpcAspNetCoreInstrumentationAddsCorrectAttributes_New(bool? enableGrpcAspNetCoreSupport)
+        {
+            var configuration = new ConfigurationBuilder()
+                .AddInMemoryCollection(new Dictionary<string, string> { [SemanticConventionOptInKeyName] = "http" })
```

Comment:
Need not be done in this PR but the tests shouldn't be using the `const string` variable `SemanticConventionOptInKeyName` directly from `src` file. Our tests. would not complain about it when someone accidentally changes the value for it in the `src` file.